### DISCUSSION TITLE
开启高速草稿模式

### DISCUSSION THREAD

**Discussion by PhotometricStereo** [sjtug/SJTUThesis#828] [灌水区]
类似overleaf 的draft高速模式：
由于章节较多，图片较多（已转为pdf）每次修改后需要30s左右编译，希望可以提高view速度。
目前发现可以使用`\documentclass[draft]{sjtuthesis}` :D
或者`\usepackage[draft]{graphicx}`实现。

**Comment by LogCreative** [comment]
- 当然也可以注释章节入口，这样只编译该章节。如果需要交叉引用可以考虑 `\include` 与 `\includeonly` 的[使用](https://github.com/CTeX-org/learnlatex.github.io/blob/zh-Hans/zh-Hans/lesson-13.md#%E4%BD%BF%E7%94%A8-include-%E5%92%8C-includeonly)。
- 如果不需要使用 TikZ，也可以注释掉加载 TikZ 的行。见 #777 
https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/setup.tex#L178-L179
